Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM golang:alpine as BUILD
+
+WORKDIR /relayer
```

Comment:
not `WORKDIR $GOPATH/src/github.com/iqlusioninc/relayer`?

Reply:
yea we can go that way. It doesn't really matter anymore where the file it, but keeping in gopath bring consistency.